client/src/components/NavLink/NavLink.css

```diff
@@ -0,0 +1,22 @@
+.NavLink {
+  background-color: var(--yellow);
+  color: var(--black);
+  font-family: var(--default-font);
+  padding: 20px;
+  position: relative;
+}
+.NavLink:hover {
+  filter: brightness(90%);
+}
+.dropdown-hide {
+  position: absolute;
+  top: -500%;
+  left: 0;
+  width: 100%;
+}
+.dropdown-show {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+}
```

Comment:
This is a good approach - I like you're thinking. A better way to do this is to have the dropdown in position and then hide and show it by setting `opacity` from `0` to `1` and changing the value of `pointer-events`. That way you don't have to move it around